shapely/geometry/linestring.py

```diff
@@ -145,5 +145,30 @@ def parallel_offset(
             distance *= -1
         return shapely.offset_curve(self, distance, resolution, join_style, mitre_limit)
 
+    def parallel_offset(self, *args, **kwargs):
+        """This method is an alias for offset_curve method.
+
+        Returns a LineString or MultiLineString geometry at a distance from
+        the object on its right or its left side.
+
+        The side parameter may be 'left' or 'right' (default is 'right'). The
+        resolution of the buffer around each vertex of the object increases by
+        increasing the resolution keyword parameter or third positional
+        parameter. Vertices of right hand offset lines will be ordered in
+        reverse.
+
+        The join style is for outside corners between line segments. Accepted
+        values are JOIN_STYLE.round (1), JOIN_STYLE.mitre (2), and
+        JOIN_STYLE.bevel (3).
+
+        The mitre ratio limit is used for very sharp corners. It is the ratio
+        of the distance from the corner to the end of the mitred offset corner.
+        When two line segments meet at a sharp angle, a miter join will extend
+        far beyond the original geometry. To prevent unreasonable geometry, the
+        mitre limit allows controlling the maximum length of the join corner.
+        Corners with a ratio which exceed the limit will be beveled.
```

Comment:
On second thought, maybe we can remove this documentation here (to not have it duplicated with the other method, since it is quite long), and just refer to `offset_curve` to see the documentation.